### PR TITLE
Add causal inference paper to publications

### PR DESCRIPTION
**Pull request by krivard**
@RoniRos / @larrywasserman check that the field values are correct for the enclosed; ~wasn't sure how to encode a paper that's been submitted but not yet published~. I took the liberty of:

* ~Uploading the pdf to our AWPS file space, so you don't need to worry about hosting it somewhere else~ (using the ArXiv preprint instead)
* Locating an appropriate hero image of the correct size and format

### CODE REVIEW

File: content/about/publications/index.md [hunk 1]
```diff
@@ -1,6 +1,12 @@
 ---
 title: Research and White Papers
 papers:
+  - title: "Causal inference in the time of COVID-19"
+    image: causal.jpg
+    authors: Bonvini, Kennedy, Ventura, Wasserman
+    link: https://publishing.andrew.cmu.edu/collections/delphi-web/publications/causal-inference-in-the-time-of-covid-19.pdf
```
Comment (capnrefsmmat) [inline content/about/publications/index.md:7]: You could also link to the preprint on arXiv: https://arxiv.org/abs/2103.04472

also is publishing.andrew.cmu.edu the final link? I've been using https://www.cmu.edu/delphi-web/

Comment (krivard) [reply content/about/publications/index.md:7]: Is the preprint link preferred? I didn't know it was on arXiv, larry and roni just sent a pdf

Comment (krivard) [reply content/about/publications/index.md:7]: Larry votes for arXiv; lemme fix

File: content/about/publications/index.md [hunk 2]
```diff
@@ -1,6 +1,12 @@
...
+    journal: (in review) Annals of Applied Statistics
```
Comment (krivard) [inline content/about/publications/index.md:8]:
```suggestion
    link: https://arxiv.org/abs/2103.04472
    journal: ArXiv preprint
```

**Comment by krivard** [comment]
Field values have been approved offline; using superpowers to merge

**Pull request merged**